contracts/package.json

```diff
@@ -4,7 +4,7 @@
     "@nomicfoundation/hardhat-toolbox": "^2.0.2",
     "@openzeppelin/contracts": "^4.9.6",
     "@openzeppelin/contracts-upgradeable": "^4.9.6",
-    "@openzeppelin/hardhat-upgrades": "^1.28.0",
+    "@openzeppelin/hardhat-upgrades": "^3.9.0",
```

Comment:
We can't bump `oz/hardhat-upgrades` due to `ethers` dependency.

Reply:
I told dependabot https://github.com/kaiachain/kaia/pull/359#issuecomment-2861933806.